CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Changed
 - [Php] Permit both PHPUnit 10 and PHPUnit 11 ([#200](https://github.com/cucumber/messages/pull/200) [ciaranmcnulty](https://github.com/ciaranmcnulty))
+### Fixed
+- [cpp] Don't rely on GitHub API for latest versions
```

Comment:
That was for some context. @kieran-ryan and I were "observing" a similar problem on gherkin and wanted to keep you informed on why the build had failed. By:
1. Not relying on api.github.com because of rate limits issues
2. Retrying downloads instead of just failing the whole build and forcing everyone to restart CI jobs or hunting for a cluprit ;)